recipe/meta.yaml

```diff
@@ -154,26 +188,28 @@ outputs:
         - sysroot_{{ ctng_target_platform }}
       run:
         # For cpp and crt{i,n}.o
-        - gcc_impl_{{ ctng_target_platform }} >=7.2.0
-        - {{ pin_subpackage("libgfortran-ng") }}
-        - {{ pin_subpackage("libgcc-ng", max_pin=None) }}
+        - gcc_impl_{{ ctng_target_platform }} >={{ ctng_gcc }}
+        - libgfortran-ng {{ ctng_gcc }}.*       # [target_platform == ctng_target_platform]
         - sysroot_{{ ctng_target_platform }}
+        # Any version newer than the devtoolset works. This can be removed when bootstrapping.
+        - libgcc-ng >=4.9
```

Comment:
While this may be technically true from an ABI perspective, for the conda packages I am not sure we can do this. At some point, we changed the layout of libgcc-ng to support swapping openmp. While this change is mostly a runtime change, we may see conflicts with older versions of libgcc-ng that had libgomp vendored.

Reply:
That's not relevant at all. Let's take `gcc_impl_linux-aarch64` running on `linux-64`. `libgcc-ng` package is a runtime dep, but that is a `linux-64` package and is useful only for running the compiler, it is not used for libraries and executables built by the compiler because the compiler is producing arm64 code. The compiler will look at `libgcc-ng` from `host` which has the correct files

Reply:
I don't think I was very clear. Let me try one more time. If we install an older libgcc-ng in the build env with the compiler when cross-compiling, I think we might get conflict errors for the build env when conda is installing the packages due to different layouts. I might not be following something here too.

Reply:
I've kept the requirements as is for native compilers, so there shouldn't be a problem.

Reply:
Ahhhhh. The cross compilers all have the new layout. SGTM and thanks for answering my questions!